.github/workflows/main.yaml

```diff
@@ -0,0 +1,38 @@
+name: main
+
+on:
+  push:
+    branches: [ main ]
+
+
+jobs:
+  build:
+    name: pipeline
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: install sbot
+        run: SEMVERBOT_VERSION=1.1.0
+          mkdir bin
+          echo "$(pwd)/bin" >> $GITHUB_PATH
+          curl -o bin/sbot -L https://github.com/restechnica/semverbot/releases/download/v$SEMVERBOT_VERSION/sbot-linux-amd64
+          chmod +x bin/sbot
+
+      - name: make venv
+        run: python -m venv venv
+
+      - name: activate venv
+        run: . venv/bin/activate
+
```

Comment:
would be good to have a `sbot update version` step as well I think

Reply:
Yeah, missing a poetry install here as well

Reply:
updated